.eslintrc.yml

```diff
@@ -0,0 +1,60 @@
+extends:
+- airbnb-base
+- next
+
+env:
+  browser: true
+  es2021: true
+  es6: true
+  jest: true
+
+globals:
+  google: readonly
+  module: readonly
+  process: readonly
+
+parserOptions:
+  ecmaVersion: 8
+  sourceType: module
+  ecmaFeatures:
+    jsx: true
+
+rules:
+  arrow-body-style: off
+  arrow-parens: off
+  func-names: warn
+  func-style: off
+  max-len: warn
+  no-bitwise: off
+  no-cond-assign:
+  - warn
+  - except-parens
+  no-confusing-arrow: off
```

Comment:
I disabled [this rule](https://eslint.org/docs/rules/no-confusing-arrow) because it goes against how Hack Reactor code is usually written, and the only justification it gives is that people might misread `=>` as `>=`, which is just silly. Honestly, some of these rules feel like they were written to help product managers who don't know JavaScript.